consensus/spos/bls/blsWorker.go

```diff
@@ -5,6 +5,8 @@ import (
 	"github.com/ElrondNetwork/elrond-go/consensus/spos"
 )
 
+const peerMaxMessagesPerSec = uint32(2)
```

Comment:
@iulianpascalau

Reply:
It will work as expected because we are taking into account the message.Peer() here (not the connected peers that send the message to self). So we will process at max (2 * num consensus peers) messages in a consensus round.
For each consensus round we need at max one message originated from a consensus peer. The value 2 took into account the possibility to receive in the same round the messages from 2 adjacent rounds.